### DISCUSSION TITLE
Colorizing the entire line in a Logs panel

### DISCUSSION THREAD

**Discussion by wz2b** [grafana/grafana#67287] [Dashboards, Panels & Visualizations]
I would like to colorize the entire log line based on level, rather than having just a little colored blip on the left side.  I realize not everybody would want this so I was thinking of making this a feature bool you can turn on and off.  I don't want to waste my time if other people would not be interested in it - can I get a quick poll here on interest?

There's a few other things I would like to do to the Logs panel as well, but all of them involve adding more panel settings.  I'm not sure how everyone would feel about that.

**Comment by ollie-campbell** [comment]
Upvote for this....

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82660. Feel free to continue the discussion around this there. Thank you!
